dotnet/samples/Concepts/ChatCompletion/OpenAI_ChatCompletionStreaming.cs

```diff
@@ -1,27 +1,36 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 
+using Microsoft.SemanticKernel;
 using Microsoft.SemanticKernel.ChatCompletion;
 using Microsoft.SemanticKernel.Connectors.OpenAI;
 
 namespace ChatCompletion;
 
-// The following example shows how to use Semantic Kernel with streaming Chat Completion
+/// <summary>
+/// These examples demonstrate the ways different content types are streamed by OpenAI LLM via the chat completion service.
+/// </summary>
 public class OpenAI_ChatCompletionStreaming(ITestOutputHelper output) : BaseTest(output)
 {
+    /// <summary>
+    ///This example demonstrates how the chat completion service streams chat using the OpenAI LLM.
+    /// </summary>
     [Fact]
-    public Task OpenAIChatStreamSampleAsync()
+    public Task StreamOpenAIChatAsync()
     {
-        Console.WriteLine("======== Open AI - ChatGPT Streaming ========");
+        Console.WriteLine("======== Open AI Chat Completion Streaming ========");
 
         OpenAIChatCompletionService chatCompletionService = new(TestConfiguration.OpenAI.ChatModelId, TestConfiguration.OpenAI.ApiKey);
 
         return this.StartStreamingChatAsync(chatCompletionService);
     }
 
+    /// <summary>
+    ///This example demonstrates how the chat completion service streams chat using the Azure OpenAI LLM.
```

Comment:
/// This example demonstrates chat completion streaming using Azure OpenAI.

Reply:
Thanks, fixed.